rpc_policy.go

```diff
@@ -34,33 +35,34 @@ var DefaultRPCPolicy = map[string]RPCEndpointType{
 	"Cluster.UnpinPath":          RPCClosed,
 	"Cluster.Version":            RPCOpen,
 
-	// PinTracker methods
+	//  PinTracker methods
 	"PinTracker.Recover":    RPCTrusted, // Called in broadcast from Recover()
 	"PinTracker.RecoverAll": RPCClosed,  // Broadcast in RecoverAll unimplemented
 	"PinTracker.Status":     RPCTrusted,
 	"PinTracker.StatusAll":  RPCTrusted,
 	"PinTracker.Track":      RPCClosed,
 	"PinTracker.Untrack":    RPCClosed,
 
-	// IPFSConnector methods
+	//  IPFSConnector methods
 	"IPFSConnector.BlockGet":   RPCClosed,
 	"IPFSConnector.BlockPut":   RPCTrusted, // Called from Add()
 	"IPFSConnector.ConfigKey":  RPCClosed,
 	"IPFSConnector.Pin":        RPCClosed,
 	"IPFSConnector.PinLs":      RPCClosed,
 	"IPFSConnector.PinLsCid":   RPCClosed,
+	"IPFSConnector.RepoGC":     RPCTrusted,
```

Comment:
I don't think we want trusted peers to be allowed to RepoGC other peers even if they are trusted. We can leave this as RPC closed. Please add comment (in `comments` variable in policygen.go)

Reply:
For Cluster.RepoGC to work it has to broadcast IPFSConnector.RepoGC. If this can't happen, there won't be cluster repoGC, only local one.

Reply:
Hmm I don't feel conformtable opening this up on follower-clusters because of the potential impact of a trusted peers mistakenly gc-ing a bunch of people. We should come with a way that users can opt-in to certain permissions but I guess it needs to stay as Trusted for the moment.

Reply:
@hsanjuan do we have a place to document the current edges cases with the different cluster ‘modes’ like the one you just mentioned? I fear it will get lost in a PR and github search is sketchy at the best of times.